yarn-project/archiver/src/archiver/kv_archiver_store/block_body_store.test.ts

```diff
@@ -0,0 +1,24 @@
+import { L2Block } from '@aztec/circuit-types';
+import { openTmpStore } from '@aztec/kv-store/utils';
+
+import { KVArchiverDataStore } from './kv_archiver_store.js';
+
+describe('Block Body Store', () => {
+  let archiverStore: KVArchiverDataStore;
+
+  beforeEach(() => {
+    archiverStore = new KVArchiverDataStore(openTmpStore());
+  });
+
+  it('Should add and return block bodies', async () => {
+    const { body } = L2Block.random(1);
```

Comment:
```suggestion
    const body = Body.random();
```
Now you can just do this ^